web/src/lib/components/admin-page/settings/admin-settings.svelte

```diff
@@ -13,20 +11,26 @@
   import type { SettingsResetOptions } from './admin-settings';
   import { t } from 'svelte-i18n';
 
-  export let config: SystemConfigDto;
+  interface Props {
+    config: SystemConfigDto;
+    children: import('svelte').Snippet<[{ savedConfig: SystemConfigDto; defaultConfig: SystemConfigDto }]>;
+  }
+
+  let { config = $bindable(), children }: Props = $props();
 
-  let savedConfig: SystemConfigDto;
-  let defaultConfig: SystemConfigDto;
+  let savedConfig: SystemConfigDto | undefined = $state();
+  let defaultConfig: SystemConfigDto | undefined = $state();
```

Comment:
The `$state` rune doesn't allow that syntax. 

<img width="1018" alt="image" src="https://github.com/user-attachments/assets/40182ebf-046e-47a4-a24a-b444a983aa49">


You either do

```
 let savedConfig: SystemConfigDto | undefined = $state();
```

or

```
 let savedConfig= $state<SystemConfigDto>();
```

I went with the first syntax because it is more explicit

Reply:
Got it, k 👍🏼